<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1e1e1e;
}
QPushButton {
    background-color: #0078d7;
    color: white;
    border-radius: 6px;
    padding: 6px;
}
QLineEdit, QTextEdit {
    background-color: #2d2d2d;
    color: #e0e0e0;
}
QProgressBar::chunk {
    background-color: #00c853;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtFilePath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecompress">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textLog">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>104</width>
      <height>85</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>29</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;File Compression Tool&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboAlgorithm">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Huffman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LZW</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
